aten/src/ATen/native/cuda/Repeat.cu

```diff
@@ -29,7 +31,11 @@ static void compute_cuda(int64_t *repeat_ptr, int64_t *cumsum_ptr, int64_t *resu
 namespace at { namespace native {
 
 Tensor repeat_interleave_cuda(const Tensor &repeat) {
-    return repeat_interleave_common<compute_cuda>(repeat);
+    Tensor output = at::empty({1});
```

Comment:
You don't need to allocate a one element Tensor here, Tensor is default constructible. Just `Tensor output`.  However, more conventionally, we typically allocate the output tensor outside of `AT_DISPATCH_INDEX_TYPES` so that we don't uselessly codegen it multiple times, so it may be worth seeing if you can hoist it out.

Reply:
The way the code is currently structured doesn't make it too easy to hoist. Consider porting this kernel to be structured, see https://github.com/pytorch/rfcs/blob/rfc-0005/RFC-0005-structured-kernel-definitions.md